tracer/src/Datadog.Trace.Coverage.collector/CoverageCollector.cs

```diff
@@ -133,7 +82,30 @@ private void OnSessionEnd(object? sender, SessionEndEventArgs e)
             _logger?.SetContext(e.Context);
         }
 
-        private void ProcessFolder(string folder, SearchOption searchOption)
+        internal void Initialize()
+        {
+            try
+            {
+                _ciVisibilitySettings = CIVisibilitySettings.FromDefaultSources();
+
+                // Read the DD_DOTNET_TRACER_HOME environment variable
+                _tracerHome = Util.EnvironmentHelpers.GetEnvironmentVariable("DD_DOTNET_TRACER_HOME");
```

Comment:
Not related to this PR, but we've been discussing about being able to get rid of this variable down the line, because the native loader sets its own env vars that we _know_ are right. I wonder if it would make sense to use those env vars here etc

Reply:
Well, I need this because I have to inject (copy to the bin folder) the `Datadog.Trace` assembly. And I need to choose the right version for the target application (.NET462, netstandard, .NET 6). So yeah, if there's another way to get the root folder where all the tracer assemblies are I'm open to change it.